docs/launch-in-aws.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Launch CyberArk Conjur In The Cloud
+layout: page
+section: get-started
+---
+
+You can easily launch a private, production-ready Conjur server in the
```

Comment:
Calling this setup "production-ready" might be overselling it a bit.

Reply:
Can you recommend a more appropriate description? What do you think about "trial-ready?"